src/lib/routes/public-invite.ts

```diff
@@ -82,7 +84,7 @@ export class PublicInviteController extends Controller {
 
     async validate(
         req: IAuthRequest<TokenParam, void>,
-        res: Response<PublicSignupTokenSchema>,
+        res: Response,
```

Comment:
Why have you removed the typing here? Doesn't having that help keep the API in sync with the schemas?

Reply:
We do not return the token because this is a public access endpoint - we just return 200 if validated. The type was wrong (left over from before)

Reply:
Aha, got it! That makes sense. Thanks for clarifying 😄